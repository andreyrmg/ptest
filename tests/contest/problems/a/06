10
10 19
10 28
6 21
8 32
5 113
7 11
5 10
10 6
5 42
2 1
