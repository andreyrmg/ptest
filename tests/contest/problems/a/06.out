2 1
5 10
10 6
7 11
6 21
10 19
5 42
8 32
10 28
5 113
